### PR TITLE
First commit from bugshoo2

### PR DESCRIPTION
**Pull request by bugshoo2**
Hello

### CODE REVIEW

**Comment by vijhn** [comment]
Approved

**Comment by andresfranc** [comment]
Approved

**Comment by MiguelALopezF** [comment]
Nice Changes :D

**Comment by juliansc95** [comment]
Like

**Comment by ghost** [comment]
Ok

**Comment by juannarvaez** [comment]
@MiguelALopez I don't think so

**Review by ghost — APPROVED** [review]

**Review by juannarvaez — APPROVED** [review]
✌️

**Review by MiguelALopezF — APPROVED** [review]
(y)

File: files/bugshoo2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Julian Mondragon
```
Comment (alvaromartinez986) [inline files/bugshoo2.txt:1]: Not commented

**Pull request merged**